10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>This code also needs some explanation. It is not obvious to a new user that the vertices of the mesh will become particles in the simulation. I'm not sure if that full explanation belongs here. It may be better to put a whole paragraph at the beginning that outlines the plan to place the vertices as particles and apply potentials that are functions of the mesh.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>